### ISSUE TITLE
[Suggestion] Colours!

### ISSUE THREAD

**Issue by ChloeDawn** [shadowfacts/SleepingBag#10]
Sleeping bags for each wool colour would be interesting.

**Comment by Prospector** [comment]
tbh I think it's nice, consistant with the vanilla bed in one colour.

**Comment by ChloeDawn** [comment]
I agree. To clarify, I made this issue before I knew about https://github.com/shadowfacts/SleepingBag/pull/9.

**Comment by shadowfacts** [comment]
I'm not sure about this. On the one hand, having different colored sleeping bags based on the wool you used to craft them would be nice, on the other hand gr8pefish's new vanilla-style red texture is nice.
